src/main/java/io/github/thebusybiscuit/slimefun4/api/events/ExplosiveToolBreakBlockEvent.java

```diff
@@ -0,0 +1,83 @@
+package io.github.thebusybiscuit.slimefun4.api.events;
+
+import io.github.thebusybiscuit.slimefun4.implementation.items.tools.ExplosiveTool;
+import org.bukkit.block.Block;
+import org.bukkit.entity.Player;
+import org.bukkit.event.Cancellable;
+import org.bukkit.event.Event;
+import org.bukkit.event.HandlerList;
+import org.bukkit.event.player.PlayerEvent;
+import org.bukkit.inventory.ItemStack;
+
+import javax.annotation.Nonnull;
+import javax.annotation.ParametersAreNonnullByDefault;
+import java.util.List;
+
+/**
+ * This {@link Event} is called when a {@link Block} is destroyed by an {@link ExplosiveTool}.
+ *
+ * @author GallowsDove
+ *
+ */
+public class ExplosiveToolBreakBlockEvent extends PlayerEvent implements Cancellable {
```

Comment:
Technically this should be *Blocks* (plural) since multiple blocks are broken. The singular could imply that this was called for every single block.